src/acceptance-test/java/org/zalando/nakadi/webservice/UserJourneyAT.java

```diff
@@ -164,6 +173,17 @@ public void userJourneyM1() throws InterruptedException, IOException {
                 .statusCode(NOT_FOUND.value());
     }
 
+    private String getUpdateEventType() throws IOException {
```

Comment:
What's the story here? Why you had to create this method? Is it because created_at should not be changed?